SpatialGDK/Source/SpatialGDK/Private/Utils/SpatialDebugger.cpp

```diff
@@ -627,37 +634,136 @@ void ASpatialDebugger::DrawDebug(UCanvas* Canvas, APlayerController* /* Controll
 
 			if (Actor != nullptr)
 			{
-				FVector ActorLocation = Actor->GetActorLocation();
-
-				if (ActorLocation.IsZero())
+				FVector2D ScreenLocation;
+				ProjectActorToScreen(Actor, PlayerLocation, ScreenLocation);
+				if (ScreenLocation.IsZero())
 				{
 					continue;
 				}
 
-				if (FVector::Dist(PlayerLocation, ActorLocation) > MaxRange)
-				{
-					continue;
-				}
+				DrawTag(Canvas, ScreenLocation, EntityId, Actor->GetName(), true /*bCentre*/);
+			}
+		}
+	}
+}
 
-				FVector2D ScreenLocation = FVector2D::ZeroVector;
-				if (LocalPlayerController.IsValid())
-				{
-					SCOPE_CYCLE_COUNTER(STAT_Projection);
-					UGameplayStatics::ProjectWorldToScreen(LocalPlayerController.Get(), ActorLocation + WorldSpaceActorTagOffset,
-														   ScreenLocation, false);
-				}
+void ASpatialDebugger::SelectActorToTag(UCanvas* Canvas)
+{
+	if (LocalPlayerController.IsValid())
+	{
+		FVector2D MousePosition;
 
-				if (ScreenLocation.IsZero())
+		if (LocalPlayerController->GetMousePosition(MousePosition.X, MousePosition.Y))
+		{
+			bool CursorChanged = false;
```

Comment:
nit: please preface all bool vars with `b` and capitalise the next character. so `bCursorChanged` and `bHit` etc.

Reply:
Sure. Done.